flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/util/KinesisConfigUtil.java

```diff
@@ -44,13 +51,18 @@
 @Internal
 public class KinesisConfigUtil {
 
-	/** Maximum number of items to pack into an PutRecords request. **/
+	/**
+	 * Maximum number of items to pack into an PutRecords request.
+	 **/
 	protected static final String COLLECTION_MAX_COUNT = "CollectionMaxCount";
 
-	/** Maximum number of items to pack into an aggregated record. **/
+	/**
+	 * Maximum number of items to pack into an aggregated record.
+	 **/
```

Comment:
These changes seem irrelevant to the issue, and in the future should be avoided as part of the main commits.
It would be ok to include a `[hotfix]` commit for these changes.

Reply:
Thanks!